src/types.js

```diff
@@ -1,12 +1,72 @@
 export const TYPES = {
   tos: {
-    name: 'Terms of service',
+    name: 'Terms of Service',
     fileName: 'terms_of_service'
   },
+  website_tos: {
+    name: 'Website Terms of Service',
+    fileName: 'website_terms_of_service'
+  },
+  user_agreement: {
+    name: 'User Agreement',
+    fileName: 'user_agreement'
+  },
+  visitor_agreement: {
+    name: 'Visitor Agreement',
+    fileName: 'visitor_agreement'
+  },
+  vendor_tos: {
+    name: 'Vendor Terms of Service',
+    fileName: 'vendor_terms_of_service'
+  },
+  tou: {
+    name: 'Terms of Use',
+    fileName: 'terms_of_use'
+  },
+  acceptableUsePolicy: {
+    name: 'Acceptable Use Policy',
+    fileName: 'acceptable_use_policy'
+  },
+  termsAndConditions: {
+    name: 'Terms and Conditions',
+    fileName: 'terms_and_conditions'
+  },
+  termsAndPrivacyPolicy: {
+    name: 'Terms and Privacy Policy',
+    fileName: 'terms_and_privacy_policy'
+  },
+  termsOfUseAndConditionsOfUse: {
+    name: 'Terms of Use and Conditions of Use',
+    fileName: 'terms_of_use_and_conditions_of_use'
+  },
+  conditionsOfUse: {
+    name: 'Conditions of Use',
+    fileName: 'conditions_of_use'
+  },
+  termsOfSale: {
+    name: 'Terms of Sale',
+    fileName: 'terms_of_sale'
+  },
   privacy: {
     name: 'Privacy Policy',
     fileName: 'privacy_policy'
   },
+  privacyFull: {
+    name: 'Full Privacy Policy',
+    fileName: 'full_privacy_policy'
+  },
+  dataPolicy: {
+    name: 'Data Policy',
+    fileName: 'data_policy'
+  },
+  dataUsePolicy: {
+    name: 'Data Use Policy',
+    fileName: 'data_use_policy'
+  },
+  privacyStatement: {
```

Comment:
In favor of merging with `privacy`, unless I misunderstand and there is a difference.